src/ducks/billsMatching/Linker/billsToOperation/operationsFilters.js

```diff
@@ -0,0 +1,147 @@
+const includes = require('lodash/includes')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.